docs/user-guide/cli-configure-scs-on-headless-linux-os.md

```diff
@@ -103,15 +160,22 @@ The following steps describe how to install and build the credential store binar
     subscription-manager repos --enable rhel-7-for-system-z-optional-source-rpms
     subscription-manager repos --enable rhel-7-for-system-z-optional-debug-rpms
     ```
-6. Install the toolset:
+6. Install the toolset by issuing the following command:
     ```
     yum install devtoolset-11
     ```
-7. After you install the toolset on RHEL V7.x, you can install Zowe CLI.
+7. Install Zowe CLI.
 
-    **Important:** The SCS is installed every time that you install or update Zowe CLI. On RHEL V7.x, ensure that the Red Hat Developer Toolset is enabled every time you install or update Zowe CLI. When you do not enable the toolset, secure credential management is not available on the system. To ensure that the toolset is enabled when you install Zowe CLI, issue the following commands instead of the standard NPM install commands. For example:
+8. [Unlock the keyring manually](#unlocking-the-keyring-manually) or [unlock the keyring automatically](#unlocking-the-keyring-automatically)to load and store credentials **[correct?]**.
```

Comment:
@adam-wolfe @zFernand0 is "to load and store creds" correct here?

Reply:
same as the previous comment 😋